sympy/integrals/tests/test_manual.py

```diff
@@ -600,6 +600,20 @@ def test_issue_23566():
     assert i == -log(4 - 2*sqrt(3)) + log(2)
     assert str(i.n()) == '1.31695789692482'
 
+def test_issue_25093():
+    a, b, x = symbols('a b x')
+    assert manualintegrate(exp(a*x**2 + b), x) == sqrt(pi)*exp(b)*erfi(sqrt(a)*x)/(2*sqrt(a))
+    assert manualintegrate(sin(a*x**2 + b), x) == \
+        sqrt(2)*sqrt(pi)*(sin(b)*fresnelc(sqrt(2)*sqrt(a)*x/sqrt(pi)) + cos(b)*fresnels(sqrt(2)*sqrt(a)*x/sqrt(pi)))/(2*sqrt(a))
+    assert manualintegrate(cos(a*x**2 + b), x) == \
+        sqrt(2)*sqrt(pi)*(-sin(b)*fresnels(sqrt(2)*sqrt(a)*x/sqrt(pi)) + cos(b)*fresnelc(sqrt(2)*sqrt(a)*x/sqrt(pi)))/(2*sqrt(a))
+    a = Symbol('a', negative=True)
```

Comment:
It is potentially confusing to change the definition of a symbol like this midway through the function. Instead it is better to use something like `an = Symbol('an', negative=True)` so it is clear that this is different from `a`.